npp.py

```diff
@@ -15,23 +15,41 @@
 ## ------- Set up our list of numbers -------
 S = [25, 7, 13, 31, 42, 17, 21, 10]
 
+# TODO: Enter your token here
+token = 'Your-Token-Here'
+
 ## ------- Set up our QUBO dictionary -------
 
 # TODO:  Add code here to define your QUBO dictionary
+def define_QUBO(S):
+    Q = {}
+    
+    return Q
+
+Q = define_QUBO(S)
 
 ## ------- Run our QUBO on the QPU -------
-# TODO:  Choose QPU parameters
-chainstrength = 1
-numruns = 10
 
 # Run the QUBO on the solver from your config file
 from dwave.system import DWaveSampler, EmbeddingComposite
+sampler = EmbeddingComposite(DWaveSampler(endpoint='https://cloud.dwavesys.com/sapi/', token=token, solver={'qpu': True}))
+
+def run_on_QPU(Q, sampler):
```

Comment:
What is the purpose of this function?

If the user is expected to tune the QPU parameters, then `run_on_QPU(..)` is really:
```
def run_on_QPU(Q, sampler, chainstrength,  numruns):
    sampleset = sampler.sample_qubo(Q, chain_strength=chainstrength, num_reads=numruns)
    return sampleset
```

Since it's essentially wrapping `sample_qubo(..)` and not executing anything else, I think this function is better off as just part of the the main executable script.